src/amo/components/CategoryHead/index.js

```diff
@@ -0,0 +1,83 @@
+/* @flow */
+import config from 'config';
+import invariant from 'invariant';
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { getCanonicalURL } from 'amo/utils';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import translate from 'core/i18n/translate';
+import type { AppState } from 'amo/store';
+import type { CategoryType } from 'amo/types/categories';
+import type { I18nType } from 'core/types/i18n';
+
+type Props = {|
+  category: CategoryType | null,
+  type: 'extension' | 'theme',
+|};
+
+type InternalProps = {|
+  ...Props,
+  _config: typeof config,
+  i18n: I18nType,
+  locationPathname: string,
+|};
+
+export class CategoryHeadBase extends React.PureComponent<InternalProps> {
+  getPageTitle() {
+    const { category, i18n, type } = this.props;
+
+    invariant(category, 'category is required');
+
+    let title;
+    switch (type) {
+      case ADDON_TYPE_EXTENSION:
+        title = i18n.gettext('%(categoryName)s – Extensions');
+        break;
+      case ADDON_TYPE_THEME:
+        title = i18n.gettext('%(categoryName)s – Themes');
+        break;
+      default:
```

Comment:
will the default ever hit if type is always defined? Maybe this could be an if / else?

Reply:
This is more future-proof.